pkgs/by-name/ei/eigenlayer-cli/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "eigenlayer-cli";
+  version = "0.4.3";
+
+  src = fetchFromGitHub {
+    owner = "NethermindEth";
+    repo = "eigenlayer";
+    rev = "v${version}";
+    hash = "sha256-VO56mMkUVeZN+YJ/cxPyZQ5dGvVDirslH4jhV5hpufQ=";
+  };
+
+  vendorHash = "sha256-DDGMprfryWj9Td4PX/j5Fsjm+bGGiDgnCXmm2IhRSMo=";
+
+  ldflags = [ "-s" "-w" ];
+  subPackages = [ "cmd/eigenlayer" ];
+
+  doCheck = false;
```

Comment:
Please specify in a comment above this line why the tests are disabled.

Reply:
Added